style.css

```diff
@@ -1 +1,347 @@
-/*  Add your styling here */
+:root {
+    --bg: #f3f0ef;
+    --text: #4c4946;
+    --yellow: #f7e259;
+    --yellowLight: #fff9de;
+    --ff-body: 'Open Sans', sans-serif;
+    --ff-secondary: 'Gwendolyn', cursive;
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+}
+
+html, body {
+    min-height: 100vh;
+}
+
+html {
+    font-size: 16px;
+}
+
+body {
+    font-family: var(--ff-body);
+    -webkit-font-smoothing: antialiased;
+    font-weight: 300;
+    color: black;
+    background: var(--bg);
+    line-height: 1.4;
+}
+
+h1, h2, h3, h4, h5 {
+    font-weight: 500;
+    margin: 0;
+}
+
+/** General  */
+a {
+    display: inline-block;
+    color: inherit;
+}
+
+nav a {
+    text-decoration: none;
```

Comment:
This can be confusing for some users with low vision or just less comfortable with modern trends, especially coupled with the removal of the hyperlink blue text. Users may not realize these are links if they don't look any different from the rest of the page.